pages/test_analytics/test_suites.md

```diff
@@ -35,6 +35,25 @@ Alternatively, you can create [scheduled builds](/docs/pipelines/scheduled-build
 
 Test Analytics reviews the test results to detect flaky tests after every test run.
 
+## Run issues
+
+<%= image "run-issues.png", alt: "Screenshot of a run with issues displaying in a list, including flaky, slow and failures." %>
+
+Test Analytics will automatically detect issues per run. For each test, we currently detect up to three issues: flaky, slow and failure, listed in the following order of urgency:
+
+- **Flaky:** [See section on detecting flaky tests](#detecting-flaky-tests).
+
+- **Slow:** Slowness is measured on the comparative performance of tests within the current run. A slow test is seen as an outlier when it is part of the slowest 1% of tests that took more than 15% of the total run time.
```

Comment:
This is haaard. Does this make sense still?

```suggestion
- **Slow:** Slowness is measured by the comparative performance of tests within the current run. A test is considered slow when it is in the slowest 1% of tests that took more than 15% of the total run time.
```

Reply:
It is right?? ChatGPT offered:
> When the slowest 1% of tests extend beyond 15% of the overall run time, our system takes charge. We automatically flag these slower tests, ensuring your testing process stays agile and efficient. 
That feels closer to what we want?

Reply:
I think our human words are far better than ChatGPT's attempt.

How about?
> Slowness is measured by the comparative performance of tests within the current run. The system automatically flags slow tests when the slowest 1% of tests take more than 15% of the overall run time.